packages/load-cfg/src/index.ts

```diff
@@ -43,7 +43,8 @@ export const loadFile = (
       file = JSON.parse(fs.readFileSync(filepath, 'utf-8'))
     }
   } catch (err) {
-    file = defaultFile
+    console.warn('There was an error loading your doczrc.js config:')
```

Comment:
please, remove `doczrc.js` because `load-cfg` is a generic package not specific!